types/gzip-size/index.d.ts

```diff
@@ -1,13 +1,18 @@
-// Type definitions for gzip-size 3.0
+// Type definitions for gzip-size 4.0
 // Project: https://github.com/sindresorhus/gzip-size
 // Definitions by: York Yao <https://github.com/plantain-00>
+//                 Jimi van der Woning <https://github.com/jimivdw>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-declare function gzipSize(input: string | Buffer, callback: (error: Error, size: number) => void): string;
-export = gzipSize;
+import * as zlib from 'zlib';
+
+declare function gzipSize(input: string | Buffer, options?: zlib.ZlibOptions): Promise<number>;
+
 declare namespace gzipSize {
-    function sync(input: string | Buffer): number;
-    function stream(): NodeJS.ReadWriteStream;
+    function sync(input: string | Buffer, options?: zlib.ZlibOptions): number;
+    function stream(options?: zlib.ZlibOptions): NodeJS.ReadWriteStream;
```

Comment:
The document said it returns `stream.PassThrough`(https://github.com/sindresorhus/gzip-size#gzipsizestreamoptions)